projects/ngx-oxford/src/lib/oxford.pipe.ts

```diff
@@ -31,9 +48,14 @@ export class OxfordPipe implements PipeTransform {
     for (let i = 0; i < trimmedValues.length - 1; i++) {
       result += `${OxfordPipe.trim(trimmedValues[i])}, `;
     }
-    result += `${OxfordPipe.trim(conjunction)} ${OxfordPipe.trim(
-      trimmedValues[trimmedValues.length - 1],
-    )}`;
+    if (!truncate) {
+      result += `${OxfordPipe.trim(conjunction)} ${OxfordPipe.trim(
+        trimmedValues[trimmedValues.length - 1],
+      )}`;
+    } else {
+      result += `${trimmedValues[trimmedValues.length - 1]}, ${truncate.trail}`;
```

Comment:
Totally optional, but it'd be nice if "trail" was optional and defaulted to "…" (unicode ellipsis, not three periods).

Reply:
Done, uses the unicode character's HTML representation